management-api-guide.md

```diff
@@ -631,10 +630,23 @@ Response, if the pre-fill URL has been accessed:
 }
 ```
 
+Below are some _ideas_ for statuses to be returned:
+
+| Status       | Description                                           | Realistic?    | Comment                 |
+| ------------ | ----------------------------------------------------- | ------------- | ----------------------- |
+| `CREATED`    | The PO exists, but nothing has been done with it yet. | ✅ Yes.       |                         |     
+| `OPENED`     | The PO URL has been accessed.                         | ✅ Yes.       |                         | 
```

Comment:
How will we be able to detect this? 🤔

Reply:
We could probably just update based on a get specific prefill request.

Reply:
The get multiple product orders request is separate, so if someone uses the get specific endpoint, it should probably be the merchant that has accessed it.

Reply:
Ah, right. Didn't know 💡 

A GET from MP would then mean we would do a status change (equivalent to PATCH), which is kinda weird, but OK I guess.

Reply:
Is it doable? If not, we can skip it, and just offer info if the PO has been submitted.

Reply:
Not the most important thing IMO, and also a little bit weird functionality. But should be doable 🤔

Reply:
Agree with ole kristian, kinda strange to update based on GET, but I also could see partners liking this status.

Reply:
Partners do ask us "Has the merchant used the link we sent them?", so there is at least _some_ need for this. If it's doable and it eliminates manual work on our side, it makes sense.

We can wish that partners ask _the merchant_ if they have clicked the link, but that doesn't change anything.